test/task_card.test.js

```diff
@@ -23,4 +23,19 @@ describe('Application launch', function () {
             assert.equal(count, 1);
         });
     });
+
+    it('shows tooltip on mouse hover', function () {
```

Comment:
please, use arrow function ( `()=>{`) for anonymous function

Reply:
I can do that, but Mocha throws an error when I try to use arrow function for `describe()`.

Reply:
Every other `describe` block in gaucho is using arrow function, so I don't understand why mocha would fail